### PR TITLE
New Cheats! 👍 

### PR DESCRIPTION
**Pull request by overtimepog**

### CODE REVIEW

**Comment by VillainsRule4000** [comment]
Have you tested the brawl cheat?

**Comment by overtimepog** [comment]
not yet, it's unconfirmed, but Ima get it confirmed once it comes out

**Comment by overtimepog** [comment]
honestly, gliz doesn't seem to keep this repo updated, villain you just wanna make a better one?

**Comment by VillainsRule4000** [comment]
You're unable able to edit react values for mb as other gamemodes.
Also, on the gamemode selection screen, use React to set the value `plus` to `true` to get the new mode.

**Comment by Sanity-Z** [comment]
> You're unable to edit react values for mb as other game modes. Also, on the gamemode selection screen, use React to set the value `plus` to `true` to get the new mode.

 👍

**Comment by VillainsRule4000** [comment]
> honestly, gliz doesn't seem to keep this repo updated, villain you just wanna make a better one?

No interest in maintaining that, honestly.

**Comment by overtimepog** [comment]
> > honestly, gliz doesn't seem to keep this repo updated, villain you just wanna make a better one?
> 
> No interest in maintaining that, honestly.

fair enough, I'll keep publishing tho

**Comment by gimmy74** [comment]
lmao on spoofblooks.js you forgot to add the color blooks 💀

**Comment by pewsie** [comment]
lol id really like some tokens hack

**Comment by VillainsRule4000** [comment]
The repo is unmaintained. Don't look HERE for hacks.

**Comment by Minesraft2** [comment]
This repo wasn't even made by the real gliz, it's a fake

**Review by jace667889 — COMMENTED** [review]
10000000000000000 gold an how to use

**Comment by Minesraft2** [comment]
> This repo wasn't even made by the real gliz, it's a fake

:0 we were wrong

**Review by VillainsRule4000 — APPROVED** [review]
good pr, will never be accepted

File: factory/setCash.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+(async () => {
```
Comment (bdsealfnr) [inline factory/setCash.js:1]: 1000000000000000

Comment (marcus11132456789987654322wedfv) [reply factory/setCash.js:1]: 11234567890

**Review by jacee773838 — APPROVED** [review]

File: factory/setCash.js (hunk 1 above)
Comment (rexie999) [reply factory/setCash.js:1]: 100000000000000000

**Review by jedeh — APPROVED** [review]

**Review by jedeh — APPROVED** [review]

**Comment by vapor69333** [comment]
how you  eqiup hacks how do u you use it?

**Comment by hacker1567** [comment]
dont know

**Pull request closed without merging**